examples/cpp/dna/README.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Helix
```

Comment:
not new in this PR, but it's a bit confusing that the example is called both `dna` and `Helix`

Reply:
helix is the better name imho. DNA sounds like we're doing actual medical visualization